odoo/addons/base/models/res_partner.py

```diff
@@ -502,7 +510,7 @@ def _commercial_sync_to_children(self):
         return res
 
     def _fields_sync(self, values):
-        """ Sync commercial fields and address fields from company and to children after create/update,
```

Comment:
> """ Sync commercial fields, address fields and Salesperson from company and to children after create/update,
>      just as if those were all modeled as fields.related to the parent """

Not really true since we don't sync the salesperson with the children.

Let's go with:
> """ Sync commercial fields, address fields from company and to children after create/update,
>      just as if those were all modeled as fields.related to the parent.
>      In addition, we also sync the Sales person from the the parent."""